main.go

```diff
@@ -103,6 +103,35 @@ var defaultWatchRepositoriesSync = []string{
 	"mender-qa",
 }
 
+// List of repositories which runs integration tests, but do not need a fresh
+// built client
+var prebuiltClientRepositories = []string{
+	"create-artifact-worker",
+	"deployments",
+	"deployments-enterprise",
+	"deviceadm",
+	"deviceauth",
+	"inventory",
+	"inventory-enterprise",
+	"integration",
+	// "mender",
+	"mender-artifact",
```

Comment:
I guess this is the reason for the new list. I would argue that `mender-artifact` cannot use a pre-compiled yocto image -> mender-artifact is part of the image.

In my understanding, only repos not in `qemuBuildRepositories` are allowed to use a pre-compiled yocto image. (this links with my previous comment, as I don't think we need two lists)